src/sync.js

```diff
@@ -237,8 +237,8 @@ class Sync {
       if (err) { log.error(err) }
       if (err && err.code === 'ENOSPC') {
         callback(new Error('The disk space on your computer is full!'))
-      } else if (err && err.message === REVOKED) {
-        callback(err)
+      } else if (err && err.status === 400) {
+        callback(new Error(REVOKED))
```

Comment:
Looking at [cozy-stack's VFS API](https://github.com/cozy/cozy-stack/blob/master/docs/files.md#patch-filesfile-id-and-patch-filesmetadata), it seems that some `PATCH /files/123` or `PATCH /files/metadata` request may trigger a `400` error code... Could this be an issue?